pubspec.yaml

```diff
@@ -7,8 +7,7 @@ publish_to: 'none'
 version: 1.0.13+16
 
 environment:
-  sdk: '>=3.4.1 <4.0.0'
-  flutter: 3.29.2
```

Comment:
Why remove flutter entry? I specifically put it into here for anyone running the project to be aware of the targeting Flutter version

Reply:
Because the default template does not have it, I guess it is not common for apps to fix the Flutter version like that. But I'll  can add it back.

Reply:
Added it back